server/test/reducer_test.js

```diff
@@ -17,7 +17,8 @@ describe('bits control logic', () => {
         title: 'Fox is unedgeguardable',
         content: 'No matter what you do, you\'ll never be able to kill a recovering Fox.',
       });
-      const newState = addBit(Map(), bit);
+      const action = {type: 'ADD_BIT', data: bit};
+      const newState = reducer(undefined, action);

```

Comment:
this looks ridiculous. i've never seen anyone actually *use* `undefined` before.

Reply:
Well, it's supposed to be if there's no initial state... if there's a better way to do this then I'm all ears :-)